Sources/ApodiniHTTP/InterfaceExporter.swift

```diff
@@ -15,8 +15,11 @@ import ApodiniNetworking
 // MARK: HTTP Declaration
 
 /// Public Apodini Interface Exporter for basic HTTP
-public final class HTTP: Configuration {
+public final class HTTP: Configuration, ConfigurationWithDependents {
+    public typealias InternalConfiguration = ExporterConfiguration
+    
     let configuration: ExporterConfiguration
+    public var staticConfigurations = [AnyDependentStaticConfiguration]()
```

Comment:
Do the static configurations have to be public? What would be the reason for making them available to the outside? Same applies to the REST exporter.

Reply:
As `ConfigurationWithDependents` is defined in `Apodini`, it has to be public to be accessible from `ApodiniHTTP` and `ApodiniREST`. Swift seems to force me to make the property public too unfortunately. Removing the property from the protocol doesn't seem like a good option, because it is being used in the protocol's extension (see [here](https://github.com/Apodini/Apodini/pull/433/files#diff-5cb1ee72a2e05fe5677b3af34c6d22f81b964f8a72a53c562f2a656719336129).